go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/bitrise-steplib/steps-xcode-build-for-simulator
 
-go 1.18
+go 1.17
```

Comment:
I changed this to `1.17` due to a known issue with `golangci-lint` preventing it from being run by Go 1.18+ (https://github.com/golangci/golangci-lint/issues/2649). This was causing me issues when trying to run the `check` step on my local dev machine. According to the [system reports](https://github.com/bitrise-io/bitrise.io/tree/master/system_reports), all the Gen2 and Linux machines have Go 1.17 installed on them. Looks like the legacy macOS machines have Go 1.16 on them though. I'm not sure if this will cause problems.

Reply:
It does not, the version here is the minimum required (https://go.dev/doc/modules/gomod-ref). 1.17 probably still works with 1.16 in practice.